cypress/fixtures/fireboltCalls/dynamicVariables.js

```diff
@@ -19,12 +19,20 @@ const accountVariables = require('./account');
 const deviceVariables = require('./device');
 const accessibilityVariables = require('./accessibility');
 const localizationVariables = require('./localization');
+const capabilitiesVariables = require('./capabilities');
+const wifiVariables = require('./wifi');
+const secondScreenVariables = require('./secondscreen');
+const metricsVariables = require('./metrics');
 
 exports.DYNAMIC_FB_CALL_VARIABLES = {
   DEFAULTS: {
     ...accountVariables.ACCOUNT_VARIABLES,
     ...deviceVariables.DEVICE_VARIABLES,
     ...localizationVariables.LOCALIZATION_VARIABLES,
+    ...capabilitiesVariables.CAPABILITIES_VARIABLES,
+    ...wifiVariables.WIFI_VARIABLES,
+    ...metricsVariables.METRICS_VARIABLES,
```

Comment:
why are we having dynamic variables in all these ?

Reply:
It not needed, removed dynamic variables.

Reply:
Since the variables are removed resolving the comment